src/main/java/org/opensearch/neuralsearch/search/util/HybridSearchResultFormatUtil.java

```diff
@@ -16,6 +16,7 @@ public class HybridSearchResultFormatUtil {
     // and OpenSearch convention is that scores are positive numbers
     public static final Float MAGIC_NUMBER_START_STOP = -9549511920.4881596047f;
     public static final Float MAGIC_NUMBER_DELIMITER = -4422440593.9791198149f;
+    public static final Float MAX_SCORE_WHEN_NO_HITS_FOUND = 0.0f;
```

Comment:
I would rather place this in score combiner class. This util is class is mainly responsible for the grammar type of formatting for the query, and this score is more related to the logic itself. See for all other methods we only use magic numbers, and real info like doc id passed as method params.